ui/app/send-v2.js

```diff
@@ -552,6 +553,47 @@ SendTransactionScreen.prototype.addToAddressBookIfNew = function (newAddress) {
   }
 }
 
+SendTransactionScreen.prototype.getEditedTx = function () {
+  const {
+    from: {address: from},
+    to,
+    amount,
+    gasLimit: gas,
+    gasPrice,
+    selectedToken,
+    editingTransactionId,
+    unapprovedTxs,
+  } = this.props
+
+  const editingTx = unapprovedTxs[editingTransactionId]
+
+  editingTx.txParams = {
```

Comment:
This is mutating props.unapprovedTxs, which comes from redux store. What does `unapprovedTxs[editingTransactionId]` looks like? If it is a single level object, we can simply make a clone using the spread operator:

```es6
const editingTx = {
    ...unapprovedTxs[editingTransactionId,
    txParams: {
        from: ethUril.addHexPrefix(from),
    },
}

Reply:
Done in e5f6466c